docker/zookeeper-image/Dockerfile

```diff
@@ -153,6 +153,12 @@ COPY opt/zkEnv.sh  $DISTRO_NAME/bin/zkEnv.sh
 WORKDIR $DISTRO_NAME
 VOLUME ["$ZOO_DATA_DIR", "$ZOO_DATA_LOG_DIR", "$ZOO_LOG_DIR"]
 
+# overwrite default zkserver.sh in here to add multiaddress.enabled
+RUN rm /$DISTRO_NAME/bin/zkServer.sh
+COPY zkServer.sh /$DISTRO_NAME/bin/
```

Comment:
where is this `zkServer.sh` coming from?

Reply:
This was left over from before you pointed out that I could set multiAddress.enabled with a jvm flag. I just removed this.